openstack/objectstorage/v1/objects/results.go

```diff
@@ -108,12 +109,59 @@ func ExtractNames(page pagination.Page) ([]string, error) {
 	}
 }
 
+// DownloadHeader represents the headers returned in the response from a Download request.
+type DownloadHeader struct {
+	AcceptRanges       string    `mapstructure:"Accept-Ranges"`
+	ContentDisposition string    `mapstructure:"Content-Disposition"`
+	ContentEncoding    string    `mapstructure:"Content-Encoding"`
+	ContentLength      int64     `mapstructure:"Content-Length"`
+	ContentType        string    `mapstructure:"Content-Type"`
+	Date               time.Time `mapstructure:"-"`
+	DeleteAt           int       `mapstructure:"X-Delete-At"`
```

Comment:
I know this is returned as an int UNIX timestamp, but maybe we could convert to `time.Time` also - what do you think?